frontend/src/pages/ReviewWizard.tsx

```diff
@@ -80,10 +80,10 @@ const ReviewWizard: React.FC = () => {
                         onClick={handleSubmit}
                         style={{ marginLeft: '20px' }}
                     >
-                        Submit
+                        Invia
                     </button>
 
-                    <p className="review-note">Once the review is submitted for this question, it cannot be changed.</p>
+                    <p className="review-note">Una volta inviata la revisione per questa domanda, non è più possibile modificarla.</p>
```

Comment:
In this case "review" has been translated to `revisione`, which is more a double-check of a document (f.i. review of a document), rather than evaluation; I would change this single word to `valutazione`.

Reply:
Fixed